src/PrimeTower.h

```diff
@@ -48,7 +48,9 @@ class PrimeTower
 
 public:
     bool enabled; //!< Whether the prime tower is enabled.
+    bool specific_brim; //<! Whether the 'brim for prime tower' option is enabled (as opposed to brim in general).
```

Comment:
If so, I would use `brim_for_prime_tower_enabled` as the name because it's more self-explanatory.

Reply:
Turns out that variable wasn't even needed any more. I'll make a new commit.